src/main/java/frc/robot/RobotContainer.java

```diff
@@ -68,12 +68,12 @@ private void configureBindings() {
     driverControllerAButton.onTrue(new SetDriveTarget(drivetrainSubsystem, 0));
     driverControllerYButton.onTrue(new SetDriveTarget(drivetrainSubsystem, 180));
 
-    operatorControllerXButton.onTrue(new FloorPickup(floorIntake));
-    operatorControllerAButton.onTrue(new FloorDrop(floorIntake));
-    operatorControllerAButton.onTrue(new DropCone(intake));
-    operatorControllerBButton.onTrue(new DropCube(intake));
-    operatorControllerXButton.onTrue(new IntakeCone(intake));
-    operatorControllerYButton.onTrue(new IntakeCube(intake));
+    operatorControllerXButton.whileTrue(new FloorPickup(floorIntake));
```

Comment:
While we are at it we should map these all to unique buttons, some buttons have multiple mappings.

Other than that it looks good